argus/backends/local/local_backend.py

```diff
@@ -0,0 +1,30 @@
+from argus.backends import base as base_backend
+from argus.backends import windows
+from argus import config as argus_config
+from argus import log as argus_log
+
+
+class LocalBackendMixin(windows.WindowsBackendMixin,
+                        base_backend.BaseBackend):
+    """ Local Backend for testing windows """
+
+    def __init__(self, name=None, userdata=None, metadata=None,
+                 availability_zone=None):
+        self._name = name
+        self._availability_zone = availability_zone
+
+        self.userdata = userdata
+        self.metadata = metadata
+
+    def setup_instance(self):
+        pass
+
+    def cleanup(self):
+        pass
+
+    @property
+    def floating_ip(self):
+        return lambda: "10.7.1.206"
```

Comment:
remove the hardcoded value here and use a config value (you have to create one if not there)